packages/calypso-products/src/plans-list.tsx

```diff
@@ -673,14 +670,7 @@ const getPlanFreeDetails = (): IncompleteWPcomPlan => ( {
 					FEATURE_SITE_ACTIVITY_LOG_JP,
 					FEATURE_SHARES_SOCIAL_MEDIA_JP,
 			  ],
-	get2023PricingGridSignupStorageOptions: () => {
```

Comment:
`Non-blocking / no response needed`:

Thanks for slowly disentangling the storage add-on from the concept of features 🎉